packages/zarathustra/skills/asylum_abci_app/behaviours.py

```diff
@@ -311,6 +357,11 @@ def act(self):
         self.context.logger.info(f"In state: {self._state}")
         while self.strategy.telegram_responses:
             msg = self.strategy.telegram_responses.pop()
+            if (msg_len := len(msg)) > TELEGRAM_MSG_CHAR_LIMIT:
+                msg = msg[:TELEGRAM_MSG_CHAR_LIMIT]
+                self.context.logger.warning(
+                    f"Shortened Telegram Message from {msg_len} to {TELEGRAM_MSG_CHAR_LIMIT} chars"
+                )
```

Comment:
yes, we should consider splitting into multiple messages. Then we can also relax the request to remain within `TELEGRAM_MSG_CHAR_LIMIT` in the `SYSTEM_PROMPT`